### ISSUE TITLE
Results from SRI KG not in answer

### ISSUE THREAD

**Issue by cbizon** [ranking-agent/strider#169]
Possibly a repeat of #165 

Query: 
```
{
    "message": {
        "query_graph": {
            "nodes": {
                "n0": {
                    "id": "NCBIGene:1017",
                    "category": "biolink:Gene"
                },
                "n1": {
                    "category": "biolink:Pathway"
                }
            },
            "edges": {
                "e01": {
                    "subject": "n0",
                    "object": "n1"
                }
            }
        }
    }
}
```

Returns no results from strider.   But in the logs in the response we see:
```
{'message': 'Received response from KP', 'url': 'https://trapi.monarchinitiative.org/query', 'request': {'message': {'query_graph': {'nodes': {'n0': {'category': 'biolink:Gene', 'is_set': False, 'id': 'NCBIGene:1017'}, 'n1': {'category': 'biolink:Pathway', 'is_set': False}}, 'edges': {'e01': {'subject': 'n0', 'object': 'n1', 'predicate': 'biolink:actively_involved_in'}}}}}, 'response': {'message': {'query_graph': {'nodes': {'n0': {'id': 'NCBIGene:1017', 'category': 'biolink:Gene', 'is_set': False}, 'n1': {'id': None, 'category': 'biolink:Pathway', 'is_set': False}}, 'edges': {'e01': {'subject': 'n0', 'object': 'n1', 'predicate': 'biolink:actively_involved_in', 'relation': None}}}, 'knowledge_graph': {'nodes': {'GO:0006977': {'category': ['biolink:NamedThing', 'biolink:BiologicalProcess', 'biolink:Occurrent', 'biolink:Pathway'], 'name': 'DNA damage response, signal transduction by p53 class mediator resulting in cell cycle arrest', 
```

i.e. the KP is returning some results and GO terms mapped to the pathway.

**Comment by cbizon** [comment]
After talking to @patrickkwang I don't think that this observation is related, but I'll mention that the things being returned bound to n1 are GO terms that we don't usually consider Pathways (i.e. the node normalizer will not call them pathways)

**Comment by patrickkwang** [comment]
May be fixed by #166. Please confirm.

**Comment by cbizon** [comment]
We're now getting results from REF KG for this query.
